docs/snippets/aggregation_arguments.md

```diff
@@ -0,0 +1,3 @@
+- `name` - name of the Aggregation object
```

Comment:
```suggestion
- `name` - name of the Aggregation
```

You can use simply `name of the aggregation` or `name of the aggregation definition`